OSDOCS-8661: Configuring IP failover content audit (#74064)
openshift/openshift-docs: modules/nw-ipfailover-virtual-ip-addresses-concept.adoc

> [id="nw-ipfailover-virtual-ip-addresses-concept_{context}"]

Reviewer: It appears this module was removed from the assembly. Is it being used somewhere else? 
https://github.com/openshift/openshift-docs/pull/74064/files#diff-99eebb2772aa83a3592ed7e92101d539b046d16a056823eb8c968ad7b94b0788L55

Author: Oops! I intended to delete this after reworking the content. Thanks for catching this!

= About virtual IP addresses

> Keepalived manages a set of virtual IP addresses (VIP). The administrator must ensure that all of these addresses:

Reviewer: ```suggestion
Keepalived manages a set of virtual IP addresses (VIP). The administrator must ensure that all of the VIP addresses meet the following requirements:
```

> * Are accessible on the configured hosts from outside the cluster.

Reviewer: ```suggestion
* Accessible on the configured hosts from outside the cluster.
```

> * Are not used for any other purpose within the cluster.

Reviewer: ```suggestion
* Not used for any other purpose within the cluster.
```

[NOTE]
====

> Each VIP in the set might be served by a different node.

Reviewer: ```suggestion
Each VIP in the set can be served by a different node.
```
Not sure if this is a pre-existing condition that we have to deal with it now ("might be"); or a condition that we are about to potentially create ("can be").
